### PR TITLE
Updating thermometer feature

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
Updating thermometer feature

### CODE REVIEW

File: source/myscript.js [hunk 1]
```diff
@@ -1,109 +1,164 @@
...
-		"<div class='c-thermometer'> \
-			<div class='c-thermometer__trigger'> \
-				<div>🙂</div> \
+
+	var thermometerHtml =
+		" <div class='c-thermometer-popup'> \
+			<div class='c-thermometer-popup__close'>×</div> \
+			<svg width='19' height='19' viewBox='0 0 19 19' fill='none' xmlns='http://www.w3.org/2000/svg'><circle cx='9.5' cy='9.5' r='9.5' fill='#166DE0' fill-opacity='0.08'/><g clip-path='url(#clip0)'><path d='M15.0172 6.43306L7.81719 13.6331L4.51959 10.3211L5.36919 9.48586L7.81719 11.9339L14.1676 5.58346L15.0172 6.43306Z' fill='#166DE0'/></g><defs><clipPath id='clip0'><rect width='13.4118' height='11.1765' fill='white' transform='translate(2.79395 3.35303)'/></clipPath></defs></svg> \
+			<h5>Thank you!</h5> \
+			<p>We appreciate your feedback on this page.</p> \
```
Comment (justinegeffen) [inline source/myscript.js:13]:
```suggestion
			<p>We appreciate your feedback.</p> \
```

File: source/myscript.js [hunk 2]
```diff
@@ -1,109 +1,164 @@
...
+		</div> \
+		<div class='c-thermometer-modal__container'> \
+			<div> \
+				<div class='c-thermometer-modal__content'> \
+					<h3>Share your thoughts</h3> \
+					<p>Your feedback helps us improve. Let us know your thoughts about this page.</p> \
```
Comment (justinegeffen) [inline source/myscript.js:19]:
```suggestion
					<p>Your feedback helps us improve. Share your thoughts with us here or join the Documentation channel on our Community server.</p> \
```

Comment (justinegeffen) [reply source/myscript.js:19]: Not sure if we can add hyperlinks here but if so, I'd like to link to the Documentation channel.

**Review by justinegeffen — COMMENTED** [review]
Some non-blocking suggestions and a question. Otherwise this is looking awesome, thank you!

**Comment by justinegeffen** [comment]
Carrie, adding you to validate the wording suggestions and, if you're happy with this, I think we can merge? :D

**Comment by cwarnermm** [comment]
One general question: as docs visitors provide ratings and feedback, where is that data stored and retrieved?

**Review by cwarnermm — COMMENTED** [review]
I've added a couple of questions inline

File: source/myscript.js [hunk 3]
```diff
@@ -1,109 +1,164 @@
...
+			<div class='c-thermometer-popup__close'>×</div> \
+			<svg width='19' height='19' viewBox='0 0 19 19' fill='none' xmlns='http://www.w3.org/2000/svg'><circle cx='9.5' cy='9.5' r='9.5' fill='#166DE0' fill-opacity='0.08'/><g clip-path='url(#clip0)'><path d='M15.0172 6.43306L7.81719 13.6331L4.51959 10.3211L5.36919 9.48586L7.81719 11.9339L14.1676 5.58346L15.0172 6.43306Z' fill='#166DE0'/></g><defs><clipPath id='clip0'><rect width='13.4118' height='11.1765' fill='white' transform='translate(2.79395 3.35303)'/></clipPath></defs></svg> \
+			<h5>Thank you!</h5> \
+			<p>We appreciate your feedback.</p> \
+		</div> \
+		<div class='c-thermometer-modal__container'> \
+			<div> \
+				<div class='c-thermometer-modal__content'> \
+					<h3>Share your thoughts</h3> \
+					<p>Your feedback helps us improve. Share your thoughts with us here or join the Documentation channel on our Community server.</p> \
```
Comment (justinegeffen) [inline source/myscript.js:19]:
```suggestion
					<p>Your feedback helps us improve. Share your thoughts with us here or join the DGW - Documentation Working Group channel on our Community server.</p> \
```

**Comment by justinegeffen** [comment]
Great question, @cwarnermm! From what I understand, this needs to be hooked up to rudderstack. I can't add Eric Nelson as a reviewer, but I suspect that's the next step.

**Comment by justinegeffen** [comment]
Oh, one thing for this @asaadmahmood, on the pages other than the home page, the widget overlaps with the "Next" button. 
Screen Shot 2021-07-12 at 20 04 01

**Comment by asaadmahmood** [comment]
@justinegeffen Cool, will update that.

File: source/myscript.js [hunk 4]
```diff
@@ -1,109 +1,164 @@
...
+
+			if (eventValue > 0) {
+				// Submit DataLayer Event
+				dataLayer.push({
+					event: 'rateThisPage',
+					eventLabel: rating,
+					eventValue: eventValue,
+					eventFeedback: currentString
+				});
+
```
Comment (Adovenmuehle) [inline source/myscript.js:95]:
```
if (typeof(rudderanalytics) !== 'undefined') {
    rudderanalytics.track("feedback_submitted", { label: rating, rating: eventValue, feedback: currentString});
}
```

Comment (cwarnermm) [reply source/myscript.js:95]: @asaadmahmood - Can you incorporate @Adovenmuehle's code update please? This update captures the user input provided into Rudderstack. Once this change has been made, we can confirm whether data is moving appropriately.

Comment (Adovenmuehle) [reply source/myscript.js:93]: @asaadmahmood I think we probably need both of these calls, not replace the `dataLayer` one. Sorry should've clarified.

Comment (asaadmahmood) [reply source/myscript.js:95]: Done @Adovenmuehle

**Comment by Adovenmuehle** [comment]
Confirmed this is sending to RudderStack correctly.

**Review by cwarnermm — APPROVED** [review]
So excited to approve and merge this user feedback enhancement!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**